gabrielmulleer/teste-oak-tecnologia
# Initial project setup with Next.js and TypeScript

## Summary

This pull request sets up the initial project structure using Next.js 14 and TypeScript.

## Changes

- Created a new Next.js project with TypeScript support
- Initialized the project with basic configuration and dependencies

## Testing

1. Clone the repository.
2. Run `npm install` to install dependencies.
3. Run `npm run dev` to start the development server.
4. Verify that the default Next.js welcome page is displayed.

## Notes

This is the foundational setup for our application and will serve as the base for future development.
